app/src/main/res/drawable/mozac_ic_warning_with_bottom_padding.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit - this needs license text at the top, look at any other xml file and copy it :)